EventHubs/src/azuredeploy_activity_logs.json

```diff
@@ -43,16 +43,20 @@
             "type": "String"
         },
         "sourceCodeRepositoryURL": {
-            "defaultValue":"https://github.com/Sumologic/sumologic-azure-function",
+            "defaultValue":"https://github.com/himanshu219/sumologic-azure-function",
```

Comment:
Should point to this repo instead.

Reply:
Hi duc changes you requested for blockblobreader is outdated (see github also shows that) probably you viewed older commit see this https://github.com/SumoLogic/sumologic-azure-function/pull/9/files#diff-aa99b6eac232114ac99e66cd8667c2f4